### PR TITLE
Add new logos to the main page

### PR DESCRIPTION
**Pull request by aguschin**
https://mlem-ai-add-logos-guvxhrdialet.herokuapp.com

close https://github.com/iterative/mlem.ai/issues/248

@iterative/websites, could you please TAL? Don't know how to break this in multiple lines.

### CODE REVIEW

**Comment by aguschin** [comment]
I think they should eventually be in multiple rows, let's say 6 logos in a row. Something like this will work just fine :) https://mlflow.org

**Comment by rogermparent** [comment]
This is what I got with a little CSS tweaking:

https://user-images.githubusercontent.com/9111807/207074920-2116c4e5-ca95-4646-a926-24aadbc5a6b0.mp4

**Comment by aguschin** [comment]
Thanks @rogermparent! Tried to adjust this a bit. 

I don't like that logos aren't centered in columns, but maybe this is over optimization attempt. WDYT?

**Comment by aguschin** [comment]
also there is an empty space on the right side, didn't figure out how to fix that 
image

**Comment by aguschin** [comment]
sorry @rogermparent, can you please help removing that empty space on the right? I tried to tweak different numbers, not with no success 😂

**Comment by aguschin** [comment]
I think centering images would help 
image

**Comment by aguschin** [comment]
ok, I've resized and compressed images, now:
```
mlem.ai (add-logos) $ ls -lh src/images/logo/                                                                                                     165ms
total 504
-rw-r--r--@  1 aguschin  staff   5.8K May  5  2022 catboost.png
-rw-r--r--   1 aguschin  staff    16K Dec 13 16:37 conda.png
-rw-r--r--   1 aguschin  staff   4.2K May  5  2022 dmlc-xgboost.png
-rw-r--r--   1 aguschin  staff   9.0K Dec 13 16:37 docker.png
-rw-r--r--   1 aguschin  staff    17K Dec 13 16:37 fastapi.png
-rw-r--r--@  1 aguschin  staff    11K Dec 13 16:50 flyio.png
-rw-r--r--   1 aguschin  staff    18K Dec 13 16:37 heroku.png
-rw-r--r--   1 aguschin  staff   3.6K May  5  2022 keras.png
-rw-r--r--@  1 aguschin  staff    13K Dec 13 16:54 kubernetes.png
-rw-r--r--   1 aguschin  staff   5.7K May  5  2022 light-gbm.png
-rw-r--r--@  1 aguschin  staff   6.4K Dec 13 16:54 numpy.png
-rw-r--r--   1 aguschin  staff   6.2K Dec 13 16:37 onnx.png
-rw-r--r--   1 aguschin  staff    19K Dec 13 16:37 pandas.png
-rw-r--r--   1 aguschin  staff    15K Dec 13 16:37 python.png
-rw-r--r--   1 aguschin  staff   4.4K May  5  2022 pytorch.png
-rw-r--r--@  1 aguschin  staff   7.9K Dec 13 16:54 rabbitmq.png
-rw-r--r--@  1 aguschin  staff    17K Dec 13 16:54 sagemaker.png
-rw-r--r--   1 aguschin  staff   4.3K May  5  2022 scikit-learn.png
-rw-r--r--   1 aguschin  staff    17K Dec 13 16:37 streamlit.png
-rw-r--r--   1 aguschin  staff   5.8K May  5  2022 tensorflow.png
```

**Comment by rogermparent** [comment]
Sorry for the late reply, and thanks for all the work you've put in this @aguschin! I'll hop back in and help finish this off.

> I don't like that logos aren't centered in columns, but maybe this is over-optimization attempt. WDYT?

 I think it was trying to do the "pad the container's left side, pad the items' right side" style of spacing before, but I do tend to go for more uniform padding/margin styles when I'm building from scratch.

> UPD: for me it sometimes takes some time to load the images. Should I maybe resize them to make them more lightweight?
> 
> UPD: looks like that. I'll see how I can make them smaller

Ah yeah, these logos were implemented with plain images instead of the gatsby-ified images, so it's even more important than usual to pre-optimize them. These logos are a case that warrants keeping as plain images, as the source files are simply tracked in git and we only intend to serve them in one res. 

Thanks for doing the work of optimizing! I'll go through and see if anything can stand to be shrunk a bit more before merging.

**Comment by rogermparent** [comment]
I think I managed to slightly improve the sizes on some of the larger images, but you did a great job!

My result:
```
4.0K    ./onnx.png
8.0K    ./catboost.png
8.0K    ./fastapi.png
8.0K    ./heroku.png
8.0K    ./keras.png
8.0K    ./light-gbm.png
8.0K    ./numpy.png
8.0K    ./pandas.png
8.0K    ./python.png
8.0K    ./pytorch.png
8.0K    ./streamlit.png
8.0K    ./tensorflow.png
12K     ./conda.png
12K     ./dmlc-xgboost.png
12K     ./docker.png
12K     ./flyio.png
12K     ./rabbitmq.png
16K     ./kubernetes.png
16K     ./sagemaker.png
16K     ./scikit-learn.png
204K    .
```

Saved about 50k total. We could probably save way more with SVGs but this will do.

Command I used: 
`mogrify -filter Triangle -define filter:support=2 -thumbnail 250 -unsharp 0.25x0.25+8+0.065 -dither None -posterize 136 -quality 82 -define jpeg:fancy-upsampling=off -define png:compression-filter=5 -define png:compression-level=9 -define png:compression-strategy=1 -define png:exclude-chunk=all -interlace none -colorspace sRGB -strip ./*`
sourced from [this article on imagemagick web compression](https://www.smashingmagazine.com/2015/06/efficient-image-resizing-with-imagemagick/).

Decided on 250 width for each because that's double the width any one logo will be (double for retina).

**Comment by rogermparent** [comment]
LGTM now so I'm going to merge this in. Thanks again @aguschin!

**Review by rogermparent — APPROVED** [review]

File: src/components/Home/Header/index.tsx [hunk 1]
```diff
@@ -47,6 +60,83 @@ const logosData: Array<{
   widthLg: number
   alt: string
 }> = [
+  {
+    widthSm: 66,
+    widthMd: 88,
+    widthLg: 136,
+    src: fastapiLogo,
+    alt: 'FastAPI logo'
+  },
```
Comment (aguschin) [inline src/components/Home/Header/index.tsx:69]: @iterative/websites, could you please help to understand how I can make these images lead to redirect to a specific docs page? E.g. here I want a link to `/doc/user-guide/serving/fastapi`. For each logo the link will be different.

Comment (yathomasi) [reply src/components/Home/Header/index.tsx:69]: Sure, we can simply wrap  tag with anchor tag <a>. But, I guess we would want some styling to make sure it shows it's clickable. I will set one up for you. I guess I will be adding add another field `link` in the above data and it will make the logo clickable.
